lib/awful/util.lua

```diff
@@ -31,8 +31,8 @@ local util = {}
 util.table = {}
 
 --- The default shell used when spawing processes.
--- @param string
-util.shell = os.getenv("SHELL") or "/bin/sh"
+-- @tfield string awful.util.shell
+util.shell = watcher._shell
```

Comment:
Uhm. This means that users cannot configure the shell used by `gears.watcher`.

Since it causes so many problems to have this in `gears`: Why isn't it in `awful` instead? That way, you do not have to move so much stuff over.

Reply:
fixed